source/getting-started/use_cases/log_analysis.rst

```diff
@@ -79,11 +79,24 @@ Expand the output to see an example found in ``/var/ossec/logs/alerts/alerts.jso
     "timestamp": "2020-07-12T15:32:41.756+0000"
   }
 
-Once generated by the :ref:`Wazuh server <wazuh_server>`, the alerts are sent to the :ref:`Wazuh indexer <wazuh_indexer>` component where they are enriched with geolocation information, stored, and indexed. Kibana can then be used to search, analyze, and visualize the data. Below is an alert as displayed in the interface:
+Once generated by the :ref:`Wazuh server <wazuh_server>`, the alerts are sent to the :ref:`Wazuh indexer <wazuh_indexer>` component where they are enriched with geolocation information, stored, and indexed. Kibana can then be used to search, analyze, and visualize the data.
 
-.. thumbnail:: ../../images/getting_started/use_cases_log_analysis.png
-   :title: Security events
-   :align: center
-   :wrap_image: No
+
+.. raw:: html
```

Comment:
This case is very subtle so I understand if you consider there's no need to modify these images, but I do see a very small black line from the header you removed.


![image](https://user-images.githubusercontent.com/79285586/141322239-35e6e7e8-2dad-43c9-aed2-aeadea75575f.png)